doc/en/user/source/configuration/status.rst

```diff
@@ -73,8 +73,8 @@ The modules tab provides a summary of the status of all installed modules in the
    
    Module Status
    
```

Comment:
Where will this documentation page be? Can you show ti using GeoSevrer doc?

Reply:
Answering myself:
https://docs.geoserver.org/stable/en/user/configuration/index.html

... looks like a good location, that say, I would update the introduction text sayign it also describes how to view the GeoServe instance status.